it/router/AccountTypeForCredIdFeature.scala

```diff
@@ -327,27 +338,51 @@ class AccountTypeForCredIdFeature extends StubbedFeatureSpec with CommonStubs {
       verify(0, getRequestedFor(urlMatching("/sa/individual/.[^\\/]+/return/last")))
     }
 
-    scenario("a user logged in through GG and has no sa and no business enrolment and no inactive enrolments and affinity group not available should get Organisation as account type") {
+    scenario("a user logged in through GG and affinity group not available should get 500 response") {
```

Comment:
it feels like some parts of the old test `a user logged in through GG and has no sa and no business enrolment and no inactive enrolments and affinity group not available should get Organisation as account type` are still valid.  I think we need a `a user logged in through GG and has no sa and no business enrolment and no inactive enrolments...` test.